### ISSUE TITLE
New Era Proposal - Molder

### ISSUE THREAD

**Issue by mudgen** [Mokens/MIPs#4]
```
Author: Nick Mudge <nick@mokens.io>
Era Name: Molder
Status: active
Start Date: 1 October 2018
Starting Mint Price: 0 ETH
Mint Increase: 0 ETH
```
*Note: A molder is a moken holder, someone who owns mokens.*

### Rational
I propose for the next era that new users will be able to mint their first moken for free, only paying the gas price. After that the price is 0.02 ETH, which is about $4 USD at the time of this writing.

After the first free moken, per user, the price is fixed at 0.02 ETH and does not increase during this moken era. The Mokens contract will enforce the mint price.

The purpose of this era is to make it super easy for new people to mint mokens and join the Mokens community. That is why the era is called Molder, which means "moken holder".

New "composable" functionality will be released on [mokens.io](https://mokens.io) at the start of this era. This means people will be able to transfer ERC721 tokens into and out of mokens and see which ERC721 tokens are owned by each moken.

A major purpose of the Mokens project is to integrate and use mokens in Ethereum games and systems. But to be useful in games and systems there have to be enough mokens available at low enough prices on exchanges, or the mint price needs to be low enough. It is hoped that an era with a free and low cost mint price will help increase the supply of low cost mokens on exchanges.

Someone could think that it would be unfair if the second era of mokens are less expensive than the first era mokens. Because Genesis mokens are mokens from the first era and are the earliest/oldest mokens and have the lowest moken ID numbers, I think they will probably retain their value (but there are no promises). From my experience the earliest and oldest collectibles retain their value.

An era distinguishes a set of mokens. Mokens from one era could be inexpensive and abundant and mokens from another era scarce, expensive and more desirable.

We hope that during this era the number of new members to the community increases dramatically. Thanks to [Andrew B Coathup](https://peepeth.com/abcoathup) and [Laurent Jacob](https://github.com/laurentjacob) and [Haley Summers](https://peepeth.com/hsummers) and others for feedback about this proposal.

Definitely leave comments below to give suggestions about this era.

**If you disagree with this New Era Proposal then please write and submit your own New Era Proposal.** ***Anybody can write one.*** **You can also select negative emojis for this issue to show your disapproval. Otherwise please show your support with positive emojis.**

The process for determining a new era is here: https://github.com/Mokens/MIPs/blob/master/MIPS/mip-3-New-Era-Proposals.md

**If you want this proposed era to be selected for voting then please support it with supporting emojis.**

**Comment by laurentjacob** [comment]
I think that limiting mokens from being created is counter-productive. I think that allowing mokens to be created for free is great and setting a limitation is also great. So what about you enforce the rule you said, but if a user wants to create more, he can simply pay some ether for it?

**Comment by laurentjacob** [comment]
The limitation could also be enforced on the contract. Should not be too hard to fetch how many moken creators there are!

**Comment by mudgen** [comment]
@laurentjacob Yes, these are good points,  I will consider updating the proposal with your suggestion, to allow people to pay after the free limit is reached.

**Comment by mudgen** [comment]
> I think that limiting mokens from being created is counter-productive. I think that allowing mokens to be created for free is great and setting a limitation is also great. So what about you enforce the rule you said, but if a user wants to create more, he can simply pay some ether for it?

You are right and I am taking your advice about this.  I changed the new era proposal so that users can have one free moken and then there is a fixed price of 0.02 ETH. Thank you for your quality input!
